sofa-ark-parent/core-impl/container/src/main/java/com/alipay/sofa/ark/container/command/AbstractCommandProvider.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.alipay.sofa.ark.container.command;
+
+import com.alipay.sofa.ark.common.util.StringUtils;
+import com.alipay.sofa.ark.spi.constant.Constants;
+import com.alipay.sofa.ark.spi.service.session.CommandProvider;
+
+/**
+ * AbstractCommandProvider, provide common features.
+ *
+ * @author joe
+ * @version 2018.07.20 11:30
+ */
+public abstract class AbstractCommandProvider implements CommandProvider {
+
+    @Override
+    public boolean validate(String command) {
+        if (StringUtils.isEmpty(command)) {
+            return false;
+        }
+
+        String[] strs = process(command);
+        command = strs[0];
+        return command.equals(getCommandPre());
```

Comment:
the command validation is a litter sample,  also command parameter should  be validated.